src/parse.cpp

```diff
@@ -1,28 +1,10 @@
 #include "parse.hpp"
 
-#include <fstream>
-#include <sstream>
-
 #include "lexer.hpp"
 
-int parse(std::string filename)
+// TODO: This return type might need to change
+std::unique_ptr<FunctionAST> parse(std::vector<Token> tokens)
 {
-
```

Comment:
Are we leaving the file intake portion squarely in the main.cpp portion of the code?

Reply:
Correct, to better reflect the flow of lex->parse->generate, I moved the file reading code out of this function